skipper/cli.py

```diff
@@ -378,6 +378,21 @@ def _push_to_registry(registry, fqdn_image):
 
 
 def _prepare_build_container(registry, image, tag, git_revision, container_context, username, password, use_cache):
+    def run(command):
+        """
+        All output generated by the container runtime during this stage should
+        not be included in stdout - we should redirect it to stderr, as the
+        user is just using skipper as a wrapper to run commands like make, run
+        etc - they don't care that as a side-effect of running their commands a
+        container is being built, and they don't want the output from that
+        build process to be included in the stdout for the commands they wrap
+        with skipper.
+
+        This allows users to do things like VERSION=$(skipper make get_version)
+        without having the build process output be included in their VERSION
+        env var.
+        """
+        return runner.run(command, stdout_to_stderr=True)
```

Comment:
I think we would like to get the output to stdout in case the user provided `--verbose`

Reply:
I don't mind doing that - but I'm not sure I understand, why would the user care about whether the build output goes to stdout/stderr when doing `--verbose`?

Reply:
Good question, the only use case I can think of is to have an option to keep getting the same output to the stdout as we get today (god knows what people are using it for).

Reply:
> to keep getting the same output

It would make sense if your goal is to build a container, e.g. in `skipper build ...`. 

But with `skipper run <something>` and `skipper make <something>` building a container is just a side-effect of trying to run <something>. 

But https://xkcd.com/1172/ so :man_shrugging:

Reply:
Anyway I don't mind doing this, so just lmk what's your final decision about this

Reply:
Not a must, just fix the pylint error and let's merge

Reply:
Okay, disabled `R:380, 0: Too many local variables (16/15) (too-many-locals)` because I think it's very annoying, lmk if you prefer I fix this in some other way